wenet/bin/export_jit.py

```diff
@@ -51,7 +53,18 @@
 
     load_checkpoint(model, args.checkpoint)
     # Export jit torch script model
+
     script_model = torch.jit.script(model)
     script_model.save(args.output_file)
 
+    # Export quantized jit torch script model
+    if args.output_quant_file:
+        quantized_model = torch.quantization.quantize_dynamic(
+            model, {torch.nn.Linear}, dtype=torch.qint8
+        )
+        print(quantized_model)
+        script_quant_model = torch.jit.script(quantized_model)
+        script_quant_model.save(args.output_quant_file)
+        print('Succeed, see {}'.format(args.output_quant_file))
+
     print('Succeed, see {}'.format(args.output_file))
```

Comment:
move this before quantize model, change the info to 
1. "Export model successfully, see {}"
2. "Export quantize model successfully, see {}"